source/docs/3/basic_network_analysis.rst

```diff
@@ -16,28 +16,28 @@ Get the data
+Download the `Roadway Block <https://opendata.dc.gov/datasets/roadway-block>`_ shared by DCGISopendata data as a shapefile.
 
 .. image:: /static/3/basic_network_analysis/images/data1.png
   :align: center
     
 For convenience, you may directly download a copy of the datasets from the
 links below:
 
-`Street_Centerlines.zip <http://www.qgistutorials.com/downloads/Street_Centerlines.zip>`_
+`Roadway_Block-shp.zip <http://www.qgistutorials.com/downloads/Roadway_Block-shp.zip>`_
 
 Data Source: [DCOPENDATA]_
 
 
 Procedure
 ---------
 
-1. Locate the downloaded ``Street_Centerlines.zip`` file in the :guilabel:`Browser` panel. Expand it and drag the ``Street_Centerlines.shp`` file to the canvas.
+1. Locate the downloaded ``Roadway_Block-shp.zip`` file in the :guilabel:`Browser` panel. Expand it and drag the ``Roadway_Block.shp`` file to the canvas.
 
   .. image:: /static/3/basic_network_analysis/images/1.png
     :align: center
   
-2. You will see a new line layer called ``Street_Centerlines`` added to the :guilabel:`Layers` panel. This layer represents each road in Washington DC. Select the :guilabel:`Identify` tool in the :guilabel:`Attributes Toolbar`. Click on any road segment to see what attributes are attached to it. There are standard attributes like road name, type etc. there is an attribute called ``DIRECTIONA``. This is an import attribute for routing as it specifies whether the segment is two-way or one-way. It contains 4 different values. ``Two Way`` for two-way streets. ``One Way (Digitizing direction)`` for one-way streets where the traffic is allowed in the direction of the line (start-point to end-point) and ``One way (Against digitizing direction)`` for one-way streets where the traffic flows in the opposite direction of the line. There is also ``Unknown`` value where we will assume two-way traffic. We will now use the information in that attribute to display an arrow on streets that are one-way.
+2. You will see a new line layer called ``Roadway_Block`` added to the :guilabel:`Layers` panel. This layer represents each road in Washington DC. Select the :guilabel:`Identify` tool in the :guilabel:`Attributes Toolbar`. Click on any road segment to see what attributes are attached to it. There are standard attributes like Route-name, Road-type etc. there is an attribute called ``SUMMARYDIR``. This is an import attribute for routing as it specifies whether the segment is two-way or one-way. It contains 4 different values. ``BD`` (Both Directions) for two-way streets. ``OB``  (Digitizing direction) for one-way streets where the traffic is allowed in the direction of the line (start-point to end-point) and ``IB`` (Against digitizing direction) for one-way streets where the traffic flows in the opposite direction of the line. There is also ``?? `` value where we will assume two-way traffic. We will now use the information in that attribute to display an arrow on one-way streets.
```

Comment:
``?? `` is not appearing correctly in the output. Try removing space after ??. You may have to escape the characters for it to appear properly.

Reply:
Updated. Now it is appearing correctly.